pkg/cmd/importcmd/import_test.go

```diff
@@ -218,9 +217,10 @@ func TestImportOptions_GetOrganisation(t *testing.T) {
 
 func TestSetPreviewNamespace(t *testing.T) {
 	t.Parallel()
-	factory := clients.NewFactory()
+	factory := fake.NewFakeFactory()
```

Comment:
@hferentschik Tada - the other changes below are because I just copied the relevant section from `TestSetPreviewNamespaceWithPermanentNs` since that test definitely worked, so better safe than sorry.

Reply:
good stuff